### DISCUSSION TITLE
SSH session manager tree

### DISCUSSION THREAD

**Discussion by carlos-n** [wavetermdev/waveterm#708] [Ideas]
Is there any plan in expanding the workspaces to something more in the line with the tree session manager with folders and subfolders as in SecureCRT, MTPutty, SuperPutty, etc.... ????
For those who have to deal with SSH connections to hundreds of servers every day, that is a must.
I think this would be a killer addition to Wave.

**Comment by chisaato** [comment]
agree! I have more than 50 ssh connections,and I still using securecrt on linux, because it's the only tool with folder/tree structure.
but securecrt is not opensource, so i am looking for another tool. Wave has done a lot
